code/modules/mob/living/carbon/carbon.dm

```diff
@@ -21,6 +21,11 @@
 	QDEL_NULL(dna)
 	GLOB.carbon_list -= src
 
+/mob/living/carbon/register_init_signals()
+	. = ..()
+
+	RegisterSignal(src, SIGNAL_ADDTRAIT(TRAIT_NOBREATH), .proc/on_nobreath_trait_gain)
```

Comment:
Could you add this code and `on_nobreath_trait_gain` to a new file?
Something like `code/modules/mob/living/carbon/init_signals.dm`

Reply:
Moved them to a new file